src/main/java/org/example/service/CustomerService.java

```diff
@@ -19,9 +19,9 @@ public Customer addCustomer(long cnic) {
         sc.nextLine();
         System.out.print("Address: ");
         String address = sc.nextLine();
-        int age = customerInputService.getAge();
         GenderType sex = customerInputService.getGender();
         String dateOfBirth = customerInputService.getDateOfBirth();
+        int age = customerInputService.getAge(dateOfBirth);
```

Comment:
as now you are not taking input from user, so this function shouldn't be in CustomerInputService

create an other class, named may be
AgeCalculaotrUtil (you can decide) and have static method calculateAge

for now use that function here, 
but that won't be needed so will tell you in the call